### ISSUE TITLE
build.sh file requires Bash rather than default Bourne shell

### ISSUE THREAD

**Issue by edguy3** [opencog/atomspace#1525]
Encountered this issue while building on Ubuntu 16.04:

```
build.sh: 6: build.sh: [[: not found
Wrong GHC version installed. Running stack setup.
stack will use a sandboxed GHC it installed
For more information on paths, see 'stack path' and 'stack exec env'
To use this GHC and packages outside of a project, consider using:
stack ghc, stack ghci, stack runghc, or stack exec

```
Source is 
../opencog/haskell/CMakeLists.txt:        COMMAND sh build.sh ${CMAKE_CURRENT_BINARY_DIR}

( Pull request with fix in process)

**Comment by linas** [comment]
I merged  #1426 which is correct; I am confused about why there is also a revert that references this... ?

**Comment by linas** [comment]
closing, I assume #1426 fixed this.
